config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
   devise_for :users
-  root 'restaurants#index'
-  resources :restaurants
+  resources :restaurants, only:
+  [
```

Comment:
Good to know. Note that I advised that they keep it explicit, over their initial inclination to omit the `only:`. They had the right instinct.